ldap/servers/slapd/slap.h

```diff
@@ -2740,4 +2742,15 @@ typedef enum _upgrade_status {
 upgrade_status upgrade_server(void);
 PRBool upgrade_plugin_removed(char *plg_libpath);
 
+/* ldapi.c */
+#if defined(ENABLE_LDAPI)
+#define LDAPI_STARTUP 0
+#define LDAPI_RELOAD 1
```

Comment:
Use an enum rather than defines.

Reply:
Had a feeling you might request enums on this one, but got side tracked.  So sure I'll switch this around.

Reply:
Trust your feelings mark, you know william will always complain in a review :)

Reply:
^---

Reply:
Yeah there were so many comments in this PR I got lost in what was being asked for.  Still being used to github I suppose...